content/en/docs/plugins/_index.md

```diff
@@ -186,12 +187,7 @@ If you are interested in authoring your own plugin, or modifying an existing plu
 
 ## Plugins
 
-Plugins authored by the Falcosecurity community are at the [plugins github repository](https://github.com/falcosecurity/plugins). The current plugins are:
-
-* **cloudtrail**: Reads Cloudtrail JSON logs from files/S3 and injects as events.
-* **dummy**: Reference plugin use to document plugins interface.
-* **dummy_c**: Like dummy, but written in C++.
-* **json**: Extracts values from any JSON payload.
```

Comment:
This is a bit minor, but I actually like listing the plugins here, as it gives a reader a flavor of what's available without having to follow the link. I don't feel super strongly about it, though.

Reply:
I think we might want to auto-generate this from the plugin registry in the future. Plus, we may also want to re-invent the https://falco.org/docs/event-sources/ section a bit, now that we have plugins. This is more a reminder for ourselves, so for now I'm gonna restore this list!